tests/system/Database/Builder/DeleteTest.php

```diff
@@ -35,4 +35,28 @@ public function testDelete()
 		$this->assertEquals($expectedBinds, $builder->getBinds());
 	}
 
+	public function testGetCompiledDelete()
+	{
+		$builder = $this->db->table('jobs');
+
+		$builder->where('id', 1);
+		$sql = $builder->getCompiledDelete();
+
+		$expectedSQL = 'DELETE FROM "jobs"'
+			. "\n" . 'WHERE "id" = 1';
```

Comment:
```suggestion
		$expectedSQL = "DELETE FROM \"jobs\"\n WHERE \"id\" = 1";
```

Reply:
It seems the space before `WHERE` is not needed.
And isn't it difficult to read `"DELETE FROM \"jobs\"\nWHERE \"id\" = 1"`?

Reply:
Yes, it's quite difficult. Maybe we can use `str_replace` just like in the previous test case?

Reply:
I pushed a commit. How about this?

Reply:
It's good to me.